dask_geomodeling/tests/test_geometry.py

```diff
@@ -391,6 +391,92 @@ def test_area_empty(self):
         self.assertEqual(0, len(result))
 
 
+class TestGeometryWktSource(unittest.TestCase):
+    def setUp(self):
+        self.projection = "EPSG:28992"
+        self.geometry = shapely_transform(
+            box(135000.5, 455998, 135001.5, 455999.5),
+            "EPSG:28992", self.projection
+        )
+        self.request = dict(
+            mode='intersects',
+            geometry=box(135000.5, 455998, 135001.5, 455999.5),
```

Comment:
Can you make this geometry such that it is intersecting the self.geometry, but the centroid of self.geometry is not ?